common/src/test/java/tachyon/security/LoginUserTest.java

```diff
@@ -57,6 +57,39 @@ public void getSimpleLoginUserTest() throws Exception {
     Assert.assertFalse(loginUser.getName().isEmpty());
   }
 
+  /**
+   * Test whether we can get login user with conf in SIMPLE mode, when custom name is provided.
+   * @throws Exception
+   */
+  @Test
+  public void getCustomLoginUserTest() throws Exception {
+    TachyonConf conf = new TachyonConf();
+    conf.set(Constants.TACHYON_SECURITY_AUTHENTICATION, "SIMPLE");
+    System.setProperty(Constants.TACHYON_SECURITY_USERNAME, "tachyon-user");
+
+    User loginUser = LoginUser.get(conf);
+
+    Assert.assertNotNull(loginUser);
+    Assert.assertEquals(loginUser.getName(), "tachyon-user");
+  }
+
+  /**
+   * Test whether we can get login user with conf in SIMPLE mode, when custom name is set to an
+   * empty string. In this case, login should return the OS user instead of empty string.
+   * @throws Exception
+   */
+  @Test
+  public void getLoginUserWhenCustomIsEmpty() throws Exception {
+    TachyonConf conf = new TachyonConf();
+    conf.set(Constants.TACHYON_SECURITY_AUTHENTICATION, "SIMPLE");
+    System.setProperty(Constants.TACHYON_SECURITY_USERNAME, "");
+
+    User loginUser = LoginUser.get(conf);
+
+    Assert.assertNotNull(loginUser);
+    Assert.assertFalse(loginUser.getName().isEmpty());
```

Comment:
Is there any way to assert that the userName is really OS user? I guess I want to make sure that, in this setting, the login module does fallback to the second choice.